pkg/vcs/repo.go

```diff
@@ -58,6 +61,15 @@ func ParseURLToRepo(rawURL string) (*Repository, error) {
 		rawURL = strings.TrimSuffix(rawURL, "?ref="+ref)
 	}
 
+	// Get sub path from path.
+	if strings.Contains(path, "//") {
+		paths := strings.Split(path, "//")
+		subPath = paths[1]
+
+		path = strings.TrimSuffix(path, "//"+subPath)
+		rawURL = strings.TrimSuffix(rawURL, "//"+subPath)
```

Comment:
How do we handle sth like https://github.com/foo/bar//one//two?

Reply:
In what cases will use this case? If it is a deep dir, it should be `https://github.com/alexcodelf/webservice//dep/sub`

Reply:
Not a valid use case. I ask because it will fail the parsing logic here. Do we have any validation to avoid this?

Reply:
updated, create template will check git source parse.